src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj

```diff
@@ -6,6 +6,11 @@
   <ItemGroup>
```

Comment:
I think on feature branch, we need to consider migration, so 2.diff.sql is required.